extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -18,15 +18,21 @@ import {createUseStyles} from 'react-jss';
 
 const sidebarClass = {
   position: 'fixed !important',
-  top: 0,
-  maxHeight: '100vh !important',
+  overflowX: 'hidden !important',
+  overflowY: 'auto !important',
+  boxSizing: 'border-box !important',
+};
+
+// User overridable styles
+const defaultSidebarStyles = {
+  color: '#000000',
+  backgroundColor: '#efefef',
   height: '100vh',
+  top: 0,
+  maxHeight: '100vh',
   maxWidth: '80vw',
```

Comment:
Removed !important from `maxHeight, maxWidth, minWidth` here.  This allows user to overwrite by specifying `amp-sidebar {` selector (which is lower specificity than class selector.

Opinons?